product-service/helpers/validateProductData.ts

```diff
@@ -0,0 +1,24 @@
+export const validateProductData = (data) => {
+  let status = true;
+  let message = [];
+  if (!data.title) {
+    status = false;
+    message.push('title field is required');
+  }
+  if (!data.description) {
+    status = false;
+    message.push('description field is required');
+  }
+  if (!data.price) {
```

Comment:
Personally I made some checks for this number values. I suppose that they need to be positive only

Reply:
Yeah, agree. Will fix it.

Reply:
Done.